pkg/gatewayserver/gatewayserver.go

```diff
@@ -487,6 +487,9 @@ func (gs *GatewayServer) Connect(
 			"schedule_downlink_late",
 			"status_public",
 			"update_location_from_status",
+			"contact_info",
```

Comment:
When solving https://github.com/TheThingsNetwork/lorawan-stack/pull/6430#discussion_r1283092849 I just added the three of them, for some reason I still have it my mind that `contact_info` should still be in there since its total deprecation would only truly happen on `v4` but that is not the case. So I'll remove it and leave only the `administrative_contact` and `technical_contact`